### DISCUSSION TITLE
Got two proofs within one minute! but only the first framed a block

### DISCUSSION THREAD

**Discussion by YangC2319** [Chia-Network/chia-blockchain#4590] [Support]
2021-05-11T06:19:54.069 harvester chia.harvester.harvester: INFO     2 plots were eligible for farming 1a7a18ce7a... Found 1 proofs. Time: 0.52859 s. Total 468 plots

2021-05-11T06:20:01.178 full_node chia.full_node.full_node: INFO      Farmed unfinished_block c41666a4eea3636d7cb7ed9de0f93fcb7f..., SP: 59

2021-05-11T06:20:45.841 harvester chia.harvester.harvester: INFO     2 plots were eligible for farming 1a7a18ce7a... Found 1 proofs. Time: 0.07779 s. Total 468 plots

(and the second do not have 'Farmed unfinished_block' line)

Why is this happening? Please Help!

Many thanks

**Comment by lvcivs** [comment]
You do not always get a reward when you find a proof. Usually you do but it's not guaranteed.

**Reply by YangC2319** [reply]
So is this about the signage point where I win did not get included into the blockchin?

**Reply by YangC2319** [reply]
how lucky I am that got 2 proofs under one minute but has 13 days of estimated win time and lose one block at the same time? :(

**Reply by lvcivs** [reply]
The details are a little over my head, to be honest, but from what I understand a proof can be better or worse than another one, and not every proof is good enough that you actually get to farm the block and receive a reward.

But yeah, super lucky to get two proofs within such a short time. Congratulations at least on the other 2 chia!

**Reply by YangC2319** [reply]
Thanks, mate. I should probably buy a lottery ticket after this x)

**Comment by momala454** [comment]
it's 1a7a18ce7a on both, isn't it the same block ?

**Reply by YangC2319** [reply]
it is the same number, but does that mean the same block? I am not sure. I do not know what that number means.
and I check the height in Chia explorer, two farmer won 4 XCH at that block.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
